device-types/Ubiquiti/UP-AI-Port.yaml

```diff
@@ -0,0 +1,68 @@
+---
+manufacturer: Ubiquiti
+model: AI Port
+slug: ubiquiti-up-ai-port
+part_number: UP-AI-Port
+u_height: 0
+is_full_depth: false
+airflow: passive
+weight: 650
+weight_unit: g
+interfaces:
+  - name: eth0
+    label: PoE IN
+    type: 1000base-tx
+    poe_mode: pd
+    poe_type: type2-ieee802.3at
+    mgmt_only: false
+  - name: eth1
+    label: PoE OUT
+    type: 1000base-tx
+    poe_mode: pse
+    poe_type: type1-ieee802.3af
+    mgmt_only: false
+front_image: true
+rear_image: true
+comments: |
```

Comment:
Thank you for your contribution,
Please reduce the amount of information in the comments property, this should only be a brief description with a link to the documentation.